extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveServerConfig.java

```diff
@@ -7,9 +7,12 @@
 public class ResteasyReactiveServerConfig {
 
     /**
-     * Set this to override the default path for JAX-RS resources if there are no
-     * annotated application classes.
+     * Set this to define the application path that serves as the base URI for all
+     * JAX-RS resource URIs provided by {@code @Path} annotations when there are no
+     * {@code @ApplicationPath} annotations defined on {@code Application} classes.
+     * <p>
+     * This value is always resolved relative to {@code quarkus.http.root-path}.
      */
-    @ConfigItem(defaultValue = "/")
+    @ConfigItem(defaultValue = "")
     String path;
```

Comment:
This needs to be Optional<String> if you have an empty default

Reply:
yes. just fixed